h/templates/layouts/base.html.jinja2

```diff
@@ -29,6 +29,35 @@
     {% endfor -%}
     {% block styles %}{% endblock %}
 
+    <link rel="apple-touch-icon" sizes="57x57"
```

Comment:
Yeesh. If @robertknight could look at this and decide which of these we actually need, that would be great. And if it still ends up being loads, I don't think a `{% for size in [57, 60, 72, 76...] %}` would go amiss here.